config/daemon/ipv6.md

```diff
@@ -10,31 +10,102 @@ Before you can use IPv6 in Docker containers or swarm services, you need to
 enable IPv6 support in the Docker daemon. Afterward, you can choose to use
 either IPv4 or IPv6 (or both) with any container, service, or network.
 
-> **Note**: IPv6 networking is only supported on Docker daemons running on Linux
-> hosts.
+> **Note**
+>
+> - IPv6 support is experimental, use it with caution.
+> - IPv6 is only supported on Docker daemons running on Linux hosts.
 
-1.  Edit `/etc/docker/daemon.json`, set the `ipv6` key to `true` and the `fixed-cidr-v6` key
-    to your IPv6 subnet. In this example we are setting it to `2001:db8:1::/64`.
+## Daemon configuration
 
-    ```json
-    {
-      "ipv6": true,
-      "fixed-cidr-v6": "2001:db8:1::/64"
-    }
-    ```
+To enable IPv6, you must edit the Docker daemon configuration file located at
+`/etc/docker/daemon.json`. Configure the following parameters:
 
-    Save the file.
+```json
+{
+  "experimental": true,
+  "ipv6": true,
+  "ip6tables": true,
+  ...
+}
+```
 
-2.  Reload the Docker configuration file.
+You can optionally also configure the `fixed-cidr-v6` key, if you want to
+assign an IPv6 subnet to the default bridge network:
 
-    ```console
-    $ systemctl reload docker
-    ```
+```diff
+{
+  "experimental": true,
+  "ipv6": true,
+  "ip6tables": true,
++ "fixed-cidr-v6": "2001:db8:1::/64",
+  ...
+}
+```
+
+After saving the configuration file, reload the Docker daemon for your
+changes to take effect:
+
+```console
+$ systemctl reload docker
+```
 
 You can now create networks with the `--ipv6` flag and assign containers IPv6
 addresses using the `--ip6` flag.
 
+## Dynamic IPv6 subnet allocation
+
+If you want dynamic IPv6 subnet allocation, you must explicitly configure the
+`default-address-pools` parameter to include:
+
+- The default address pool values
+- One or more custom IPv6 supernets
```

Comment:
> it would seem a better UX if a separate setting appended pools instead of requiring users to maintain the provided defaults

Yes it could make users' life easier, but that's not where we are now :slightly_smiling_face: Feel free to open a enhancement request on moby/moby.